gufe/protocols/protocol.py

```diff
@@ -91,23 +91,22 @@ class Protocol(GufeTokenizable):
         Corresponding `ProtocolResult` subclass
 
     """
-
+    _settings: Settings
     result_cls: type[ProtocolResult]
 
-    def __init__(self, settings=None):
-        """Create a new `Protocol` instance.
+    def __init__(self, settings: Settings):
```

Comment:
`Settings` now not optional, I couldn't think of a case (besides lazy test writing, which I'm fixing here) when we'd have a Settings-less Protocol